DonjinKrawler-Client/src/main/java/donjinkrawler/Player.java

```diff
@@ -52,16 +55,22 @@ public void move(List<Wall> walls, List<Door> doors, List<Obstacle> obstacles, L
         if (isCollidingWithImmovableObject(walls) || isCollidingWithImmovableObject(decorations)) {
             return;
         }
-        data.setX(data.getX() + dx);
-        data.setY(data.getY() + dy);
+        if (backwards) {
+            commander.undo();
+        }else {
```

Comment:
Formatting. Just go through the changes and format the code

Reply:
fixed